torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19746,6 +19746,26 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            # vmap does not support inplace views
            check_inplace_batched_forward_grad=False,
            sample_inputs_func=sample_inputs_transpose_swapdims),
+    OpInfo('transpose_copy',
+           aliases=('swapdims', 'swapaxes'),
```

Comment:
I don't think these apply, and thats probably why the jit alias mapping tests are failing.

Reply:
Removed! And also removed one of the skips.  The other one continues to occur, and is in the same class as in the `squeeze_copy` decomposition pull request, https://github.com/pytorch/pytorch/pull/130941